spring-cloud-stream-binder-jms-common/src/main/java/org/springframework/cloud/stream/binder/jms/config/JmsBinderConfigurationProperties.java

```diff
@@ -0,0 +1,20 @@
+package org.springframework.cloud.stream.binder.jms.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * @author Donovan Muller
+ */
+@ConfigurationProperties("spring.cloud.stream.binders.jms")
+public class JmsBinderConfigurationProperties {
+
+    private boolean transacted = true;
```

Comment:
This is the correct default for a `DefaultMLC` but the config metadata says the default is false.

That said, it's pretty dangerous to set it to false with a DLMC (message loss) so if the container type is not configuarable, I am not sure we should expose this as an option.